public/javascripts/app/lib/icons/index.js

```diff
@@ -1,18 +1,18 @@
 import angular from 'angular'
 
-import clockActive      from './svg/clock-active.svg!text';
-import clockDisabled    from './svg/clock-disabled.svg!text';
-import infoActive       from './svg/info-active.svg!text';
-import infoDisabled     from './svg/info-disabled.svg!text';
-import publishActive    from './svg/publish-active.svg!text';
-import publishDisabled  from './svg/publish-disabled.svg!text';
-import previewActive    from './svg/preview-active.svg!text';
-import arrowDown        from './svg/arrow-down.svg!text';
-import composerIcon     from './svg/composer-icon.svg!text';
-import wrenchActive     from './svg/wrench-active.svg!text';
-import wrenchDisabled   from './svg/wrench-disabled.svg!text';
-import expandActive     from './svg/expand-active.svg!text';
-import expandDisabled   from './svg/expand-disabled.svg!text';
+import clockActive      from '!!raw-loader!./svg/clock-active.svg';
```

Comment:
The funny syntax is to drop the existing SCSS loader and as such bring in the file without conversion to a data URL